release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,82 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
 
 For more information about the Ironic provisioning service, see xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc[Deploying installer-provisioned clusters on bare metal].
 
+[id="ocp-4-12-agent-based-install-asset-input"]
+==== Agent-based installation supports two input modes
+The Agent-based installation supports two input modes:
+
+**Preferred**
+
+* `install-config.yaml` file
+* `agent-config.yaml` file
+
+**Optional**
+
+* Zero Touch Provisioning(ZTP) manifests
+
+With the preferred mode, you can configure the `install-config.yaml` file and specify Agent-based specific settings in the `agent-config.yaml` file.
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[About the Agent-based Installer].
+
+[id="ocp-4-12-agent-based-install-fips"]
+==== Agent-based installation supports FIPs compliance
+Agent-based {product-title} Installer supports {product-title} clusters in Federal Information Processing Standards (FIPS) compliant mode. You must set the value of the `fips` field to `True` in the `install-config.yaml` file.
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#agent-installer-configuring-fips-compliance_preparing-to-install-with-agent-based-installer[About FIPS compliance].
+
+[id="ocp-4-12-agent-based-install-disconnected"]
+==== Deploy an Agent-based {product-title} cluster in a disconnected environment
+You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the ` install-config.yaml` must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
+For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.html[Understanding disconnected installation mirroring].
+
+[id="ocp-4-12-agent-based-install-dual-stack"]
+==== Agent-based installation supports single and dual stack networking
+You can create the agent ISO image with the following IP address configurations:
+* IPv4
```

Comment:
This list formatting seems to be wrong.
![image](https://user-images.githubusercontent.com/619423/211771559-f9e10388-9d5d-4e65-a8e9-7c226111acb4.png)

Reply:
yes correct, rectified